src/pre.js

```diff
@@ -94,13 +95,16 @@ var glpkPromise = new Promise(function (resolve) {
 					housekeeping(P);
 					return FS.readFile(name, { encoding: 'utf8' });
 				},
-				'solve': function (lp, options) {
+				'solve': function (lp, opt) {
 					
-					const settings = _.defaults(options, {
-							msgLev: 1,
-							tmLim: 120000,
-							mipGap: 0
-					})
+					const opt_ = opt || lp.options || {};
+					
+			    const options = {
```

Comment:
the indentation here does not seem right

Reply:
hmm it was a combination of tabs & whitespace that messed up